frontend/src/app/core/services/team.service.ts

```diff
@@ -46,7 +87,7 @@ export class TeamService {
         return this.httpService.putRequest<ITeam>(`${this.routePrefix}`, team);
     }
 
-    public uploadLogo(data: FormData, id: number) {
-        return this.httpService.putRequest<IImagePath>(`${this.routePrefix}/uploadlogo/${id}`, data);
+    public uploadLogo(data: FormData, id: number | undefined) {
```

Comment:
Why do we need undefined here?

Reply:
I use this method for both team creation or update. When team is created, it doesn't have number id, so this parameter is optional

Reply:
Yes) I will fix it soon

Reply:
Fixed by making optional parameter